code/_onclick/other_mobs.dm

```diff
@@ -205,6 +205,15 @@
 	if(user.pulling != src)
 		if(!lying_attack_check(user))
 			return FALSE
+	
+	if(isseelie(src))
+		if(user.patron.type == /datum/patron/inhumen/graggar)
+			if(user.pulling == src)
+				if(user.grab_state == GRAB_AGGRESSIVE)
+					visible_message(span_danger("[user] is putting [src] in their mouth!"), \
+									span_userdanger("[user] is putting me in their mouth!"))
+					if(do_after(user, 5 SECONDS, target = src))
```

Comment:
If anything make it take as long as wing ripping, if not more since gibbing them would make them unrevivable

Reply:
how long is ring wipping

Reply:
![image](https://github.com/user-attachments/assets/58d95b84-bea2-4ae9-8bae-1a9c8b924e97)